apps/identity/src/routes/api/v1/username/[username]/quit/+server.ts

```diff
@@ -0,0 +1,22 @@
+import { error, json } from "@sveltejs/kit";
+import { getUsername, updateUsername } from "warehouse";
+import type { RequestHandler } from '@sveltejs/kit';
+import { tokenOnly } from "$lib/middlewares";
+
+export const POST: RequestHandler = async ({ params, request }) =>
+  tokenOnly(request, async () => {
+    if (!params.username) throw error(400);
+
+    const username = await getUsername(params.username);
+    if (!username) throw error(404);
+
+    const updatedUsername = await updateUsername(username.name, {
+      firstJoin: username.firstJoin,
+      lastSeen: username.lastSeen,
+      isOnline: false
+    })
```

Comment:
```suggestion
    const updatedUsername = await updateUsername(username.name, {
      isOnline: false
    })
```
Aqui só precisa atualizar os valores necessários